icons/radical.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3 12h4l3 8 3-16h7" />
```

Comment:
I think we should add an extra pixel of height to the bottom both to better match `activity` and to adjust the centre of gravity vertically:
```suggestion
  <path d="M3 12h4l3 9 4-17h7" />
```